src/coreclr/jit/lsrabuild.cpp

```diff
@@ -892,6 +892,10 @@ regMaskTP LinearScan::getKillSetForCall(GenTreeCall* call)
     if (!compiler->compFloatingPointUsed)
     {
         killMask &= ~RBM_FLT_CALLEE_TRASH;
+
+#if defined(TARGET_XARCH)
+        killMask &= ~RBM_MSK_CALLEE_TRASH;
```

Comment:
Given that this method is hot, I am wondering if we should extract something like `RBM_FLT_MSK_CALLEE_TRASH` for xarch and so we won't have to do another operation of ` &= ~` twice?

Reply:
`killMask &= ~(RBM_FLT_CALLEE_TRASH | RBM_MSK_CALLEE_TRASH)` seems fine for now.